src/common/base/ConcurrentLRUCache.h

```diff
@@ -0,0 +1,130 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef COMMON_BASE_CONCURRENTLRUCACHE_H_
+#define COMMON_BASE_CONCURRENTLRUCACHE_H_
+
+#include "base/Base.h"
+#include "base/StatusOr.h"
+#include <boost/compute/detail/lru_cache.hpp>
+#include <gtest/gtest_prod.h>
```

Comment:
why include `gtest`?

Reply:
Because we want to use marco "FRIEND_TEST"